isisd/isis_pfpacket.c

```diff
@@ -122,7 +123,9 @@ static int open_packet_socket(struct isis_circuit *circuit)
 	struct sockaddr_ll s_addr;
 	int fd, retval = ISIS_OK;
 
-	fd = socket(PF_PACKET, SOCK_DGRAM, htons(ETH_P_ALL));
+	fd = vrf_socket(PF_PACKET, SOCK_DGRAM, htons(ETH_P_ALL),
+			circuit->interface->vrf_id, circuit->interface->name);
```

Comment:
shouldn't the last parameter be the VRF name instead?

Reply:
Yes, you are correct. 
This fix is done. 
Please have a look. 

Thanks,